moduleframework/common.py

```diff
@@ -322,9 +318,28 @@ def __init__(self, *args, **kwargs):
         self.arch = None
         self.dependencylist = {}
         self.moduledeps = None
+        self.is_it_module = False
         # general use case is to have forwarded services to host (so thats why it is same)
         self.ipaddr = trans_dict["HOSTIPADDR"]
         trans_dict["GUESTARCH"] = self.getArch()
+        self.loadconfig()
+
+    def loadconfig(self):
+        """
+        Load configuration from config.yaml file.
+
+        :return: None
+        """
+        self.config = get_config()
+        doc_name = ['modularity-testing', 'meta-test-family','meta-test']
```

Comment:
Please add space after 'meta-test-family'. Me is digging.

Reply:
I'll add